commander/src/commands/core/upgrade.ts

```diff
@@ -174,7 +175,11 @@ export default class UpgradeCommand extends BaseCommand {
 									cacheDir,
 								);
 
-								await extract(fileDir, fileName, installationPath);
+								await tar.x({
```

Comment:
why not using `extract` now?

Reply:
tar has the extract function directly.. we dont need an extra wrapper just for calling it

Reply:
I think you already defined the extra wrapper, and probably it would be easier if we need to change library (like this case) if we keep the wrapper

Reply:
ok ll move it back.